synctool/src/test/java/org/duracloud/sync/endpoint/DuraStoreChunkSyncEndpointTest.java

```diff
@@ -173,6 +172,7 @@ public void testAddNewContent() throws Exception {
     }
 
     @Test
+    @Ignore("move to ChunkingContentStoreImplTest?")
```

Comment:
I still need to go through these ignored tests to understand them a bit better, but I was thinking they are likely better handled in the `storeclient` now with some of the other tests. I just haven't been satisfied with the extensive amount of mocking I've had to do to get some of the tests to work. I might try a stub class to make it easier but haven't had the opportunity to dig further yet.